### PR TITLE
to_report param

### LINKED ISSUE #32: Add argument for what pieces of messages, output, and result to be included in log

### PR DESCRIPTION
**Pull request by kodesiba**
Update axecute and log_write with to_report param, closes #32

### CODE REVIEW

**Review by bms63 — COMMENTED** [review]
Do we want the add-in to have these options reflected in it?  Like a user could toggle these on/off with checkboxes?

**Comment by kodesiba** [comment]
> Do we want the add-in to have these options reflected in it? Like a user could toggle these on/off with checkboxes?

Added these as a group of checkboxes that we can extend as we need other things, also added a waiter to the addin since I didn't know when the file was done when testing

**Comment by bms63** [comment]
Hey Nathan - I increased the size of the window so you don't have to scroll and put a fancy solar system spinner in there...that waiter package is really cool!

**Review by thomasp-b — APPROVED** [review]
Works like a charm!

File: R/timberAddin.R [hunk 1]
```diff
@@ -5,10 +5,12 @@
 #' @importFrom shiny textInput checkboxInput actionButton uiOutput reactiveValues observeEvent renderText stopApp runGadget conditionalPanel fluidRow column tags HTML
 #' @importFrom rstudioapi selectFile selectDirectory
 #' @importFrom stringr str_locate_all str_replace
+#' @importFrom waiter useWaiter waiter_show waiter_hide spin_fading_circles
```
Comment (nicholas-masel) [inline R/timberAddin.R:8]:
```suggestion
#' @importFrom waiter useWaiter waiter_show waiter_hide spin_fading_circles spin_solar
```

Comment (bms63) [reply R/timberAddin.R:8]: Thanks Nick!

**Pull request merged**